HW1/P6/P6.py

```diff
@@ -0,0 +1,78 @@
+
+# coding: utf-8
+
+# In[1]:
+
+import findspark
+findspark.init()
+from collections import Counter
+import pyspark
+sc = pyspark.SparkContext(appName="Spark1")
+import numpy as np
+
+
+# In[2]:
+
+full_text = sc.textFile("./pg100.txt")
+#clean data
+cleaner = lambda word: False if (word.isupper() or word.isdigit() or word == u'' or (word[:-1].isupper() and word[-1] == u'.')) else True
+full_text = full_text.flatMap(lambda line: line.split(u" ")).filter(cleaner)
+
+#get list of words in order
```

Comment:
Wow you did this in a completely different way than I did. It's interesting how there are so many different ways to do the same thing.